src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -170,11 +174,17 @@ def _list_devices(self, cmd):
         else:
             # Return a human readable version
             result = ""
+
             for inventory_node in completion.result:
-                result += "{0}:\n".format(inventory_node.name)
+                result += "Host {0}:\n".format(inventory_node.name)
+
+                if inventory_node.devices:
+                    result += inventory_node.devices[0].pretty_print(only_header=True)
```

Comment:
why only the first device?

Reply:
I think is more readable to have one header line starting the list of devices for each host.